test_utils/tasks.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tasks used in tests
```

Comment:
pytest [recommends](http://doc.pytest.org/en/latest/goodpractices.html#choosing-a-test-layout-import-rules) omitting `__init__.py` from test directories for a few different reasons, but without it pylint won't analyze anything in the directory (at least not without separately running it on each contained module and package).